### PR TITLE
Add MariaDB plugin

### PR DESCRIPTION
**Pull request by m1ch**
Thank you for your submission to the FreeNAS community plugins repository!

Please ensure the following guidelines are met when submitting a new Plugin.

1. Add the entry to INDEX in alphabetical order - Yes
2. Includes a new icon in the ./icon/ directory - Yes
3. Ensure the submitted icon file is a valid PNG that is 128x128 in size - Yes
4. Add new plugin cirrus task - Yes

### CODE REVIEW

File: mariadb.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "MariaDB",
+  "plugin_schema": "2",
+  "release": "12.2-RELEASE",
+  "artifact": "https://github.com/m1ch/iocage-plugin-mariadb.git",
+  "official": false,
+  "properties": {
+    "allow_raw_sockets": "1",
```
Comment (sonicaj) [inline mariadb.json:8]: Please change the values `1/0` to integer

**Review by sonicaj — COMMENTED** [review]
Thank you for your contribution! Just one minor nit

File: mariadb.json (hunk 1 above)
Comment (m1ch) [reply mariadb.json]: Thanks for the hint. Update done!

**Review by sonicaj — APPROVED** [review]
Thanks!

**Pull request merged**